include/swift/AST/AnyRequest.h

```diff
@@ -150,6 +150,9 @@ class AnyRequest {
     other.storageKind = StorageKind::Empty;
   }
 
+  AnyRequest(const AnyRequest &&other)
+    : storageKind(other.storageKind), stored(other.stored) { }
+
```

Comment:
A `const &&` move constructor is useless, and probably a typo. It looks like it can be removed and left as default as well.

Reply:
@jckarter yeah, that's what I thought too ... and then realized that they actually made it do something that doesn't make sense *shrug*.  I'll happily remove it.